homeassistant/components/html5/issues.py

```diff
@@ -0,0 +1,46 @@
+"""Issues utility for HTML5."""
+
+import logging
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.issue_registry import IssueSeverity, async_create_issue
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+SUCCESSFUL_IMPORT_LOG = """Loading HTML5 push notification via configuration.yaml is deprecated,
+ it has been automatically imported. Once you have confirmed correct
+ operation, please remove HTML5 notification configuration from
+ configuration.yaml""".replace("\n", "")
+SUCCESSFUL_IMPORT_TRANSLATION_KEY = "deprecated_yaml"
+
+FAILED_IMPORT_LOG = """Loading HTML5 push notification via configuration.yaml is deprecated and
+ the automatic import has failed. Please remove the HTML5 notification configuration from configuration.yaml
+ and setup HTML5 from scratch via the UI.""".replace("\n", "")
+FAILED_IMPORT_TRANSLATION_KEY = "deprecated_yaml_import_issue"
+
+
+def async_create_html5_issue(hass: HomeAssistant, import_success: bool) -> None:
+    """Create issues for HTML5."""
+    _LOGGER.warning(SUCCESSFUL_IMPORT_LOG if import_success else FAILED_IMPORT_LOG)
+
+    translation_key = (
+        SUCCESSFUL_IMPORT_TRANSLATION_KEY
+        if import_success
+        else FAILED_IMPORT_TRANSLATION_KEY
+    )
```

Comment:
Please convert multi-line ternaries into if blocks or rework so its not multi-line

Reply:
done